ResistanceInAWire/resistance_in_a_wire_algorithm.py

```diff
@@ -0,0 +1,61 @@
+""" This contains an algorithm module that find the resistance in a wire
+ using functional programming. """
+
+# Define a lamba function to calculate the resistance of a wire using resitivity as rho,
+# length as l and cross-section area of wire as A.
+
+calculate_resistance = lambda rho, length, area: rho * length / area
```

Comment:
ResistanceInAWire/resistance_in_a_wire_algorithm.py:7:23: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)

As I mentioned in class, it works but is highly discouraged, mostly if imported as module it is not clear that this is a function and cannot be used.
Better would be in line 21:
```python
  yield lambda rho, length, area: rho * length / area
```

Reply:
Ive tried this and pylint still says the lambda is unnecessary Ive added a work around with the and apply_formula and the pylint still says the lambda is not necessary but the score is higher.